nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -77,14 +81,13 @@ in
       enable = mkDefault true;
     };
 
-    # TODO: disable this once k3s supports cgroupsv2, either by docker
-    # supporting it, or their bundled containerd
-    systemd.enableUnifiedCgroupHierarchy = false;
```

Comment:
Ok. I am not sure about this bit anymore.

Reply:
As of [this PR](https://github.com/NixOS/nixpkgs/pull/114215), k3s does work with unified cgroups, though you also have to get a flag right (I think just making the kubelet cgroup driver of systemd)

I've been meaning to make a separate PR to change it over to defaulting to unified cgroups.

I think it'll require release notes since it's a breaking change, so it's probably worth keeping isolated in its own PR separate from this one.